src/ed_object_models/conversion_sdf.py

```diff
@@ -355,17 +355,12 @@ def parse_to_xml(xml, item, list_name=""):
     elif isinstance(item, dict):
         for k, v in item.items():
             # attributes
-            if k == "name" or k == "version" or k == "type":
+            attributes = ["version", "type"]
```

Comment:
This file is, according to @MatthijsBurgh, the file that generates all the sdf files. So the changes in the remaining 308 files are purely due to the changes in this file.

Reply:
I find this code in general hard to read and to understand, i.e., you add stuff from an input to an XML element but do not explain how that should be done. I can imagine the change: you don want to add the 'name' attribute if you're include something. I guess a complete clarification would be outside of the scope of this MR and probably not the most important thing to fix.